packages/playwright/src/reporter.ts

```diff
@@ -327,9 +340,21 @@ class ReplayPlaywrightReporter implements Reporter {
   async onEnd() {
     try {
       await this.reporter.onEnd();
-      if (!this._foundReplayBrowser) {
+
+      const projectsWithoutReplay = Object.keys(this._projects).filter(projectName => {
+        const { executed, usingReplay } = this._projects[projectName];
+        return executed && !usingReplay;
+      });
```

Comment:
I'm not sure this is the check we want here. Shouldn't we _actually_ be checking to ensure that at least one project using Replay ran? (It's okay if others ran that weren't using it.) e.g.

```js
const shouldWarn = !Object.keys(this._projects).find(projectName => {
  const { executed, usingReplay } = this._projects[projectName];
  return executed && usingReplay;
});
```

I don't think we'd want to show ourselves this warning in the case when we chose to run with both regular Chromium and Replay Chromium, would we? (Maybe you think we would? Maybe you're worried about a case where different projects have different sets of tests they run? But that still seems pretty intentional to me.)

Reply:
When I initially implemented this warning I specifically didn't want to warn when "mixed" projects would be executed... and I just forgot about this intention when working on this. I think you are right here. This also addresses the other concern that you raised